src/renderer/remote-loader.ts

```diff
@@ -125,8 +125,11 @@ export class RemoteLoader {
       const values: EditorValues = {};
 
       for (const [id, data] of Object.entries(gist.data.files)) {
+        const content = data.truncated
+          ? await fetch(data.raw_url).then((r) => r.text())
```

Comment:
> From gist docs:
> If you need the full contents of the file, you can make a GET request to the URL specified by raw_url. Be aware that for files larger than ten megabytes, you'll need to clone the gist via the URL provided by git_pull_url.

Would it be better to use git_pul_url? 🤔

Reply:
Wouldn't that require cloning the repo, which would add significant complexity? It's a good point though, would be good to check for that case and throw an error saying the gist is too large to load. I think it's reasonable for Fiddle to not handle that specific case since it's unlikely to ever come up. 🙂

Reply:
Oh, sorry. I thought `git_pull_url` is equivalent to `raw_url. 😅